### ISSUE TITLE
run pod sandbox failed

### ISSUE THREAD

**Issue by alexvirtualbr** [kubernetes-sigs/cri-o#1937]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
**Description**

FATA[0001] run pod sandbox failed: rpc error: code = Unknown desc = write child: broken pipe

**Steps to reproduce the issue:**
1. When i tried run POD_ID=$(sudo crictl runp test/testdata/sandbox_config.json)

**Describe the results you received:**

FATA[0001] run pod sandbox failed: rpc error: code = Unknown desc = write child: broken pipe

**Describe the results you expected:**

I expected receive something the same your describe in https://github.com/kubernetes-sigs/cri-o/blob/master/tutorial.md:

ID: cd6c0883663c6f4f99697aaa15af8219e351e03696bd866bc3ac055ef289702a
Name: podsandbox1
UID: redhat-test-crio
Namespace: redhat.test.crio
Attempt: 1
Status: SANDBOX_READY
Created: 2016-12-14 15:59:04.373680832 +0000 UTC
Network namespace: /var/run/netns/cni-bc37b858-fb4d-41e6-58b0-9905d0ba23f8
IP Address: 10.88.0.2
Labels:
	group -> test
Annotations:
	owner -> hmeng
	security.alpha.kubernetes.io/seccomp/pod -> unconfined
	security.alpha.kubernetes.io/sysctls -> kernel.shm_rmid_forced=1,net.ipv4.ip_local_port_range=1024 65000
	security.alpha.kubernetes.io/unsafe-sysctls -> kernel.msgmax=8192

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `crio --version`:**

crio version 1.12.0-dev
commit: "6adb5616c2e78f9095f1c81e002fa6aa84bc5a82"

**Comment by ccollicutt** [comment]
Just FYI, in my case I found that crio was looking for runc in the wrong place. The tutorial does say to put it in /usr/bin so this is an error on my side. In my case initially runc came from the projectatomic:ppa, which puts it into /usr/sbin, then I downloaded a new version of runc per the instructions but did not put it into /usr/bin, instead mistakenly /usr/sbin. Maybe check that?

```
crio[27009]: [conmon:e]: Runtime path /usr/bin/runc is not valid No such file or directory
```
